config/lib/app.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+var config = require('../config'),
+	mongoose = require('./mongoose'),
+	express = require('./express'),
+	chalk = require('chalk');
+
+// Initialize Models
+mongoose.loadModels();
```

Comment:
Thank you. That makes sense. 

Could one of those scenario to use the exported function, be loading the models for the tests? Forgive me, I haven't spent much time with the test coverage, but I am trying to learn :)